torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -70,7 +70,7 @@ void flag_to_float(const T* src, float* dst, int64_t n) {
   }
 }
 
-template <typename T>
+template <typename T, std::enable_if_t<std::is_same<T, bool>::value || std::is_same<T, uint8_t>::value, bool> = true>
```

Comment:
Weird that UT doesn't expose such an error. The compiler turns out to have an inconsistent lookup order for the pointer data types? Anyway, doing SFINAE is safer...

Reply:
Oh, an offline chat with @XiaobingSuper , compiler does things right but the checks in UT are weak.